lib/yt/resource.rb

```diff
@@ -51,7 +51,7 @@ def self.has_attribute(name, options = {}, &block)
         keys = Array(options[:in]) + [name]
         part = keys.shift
         value = @data[part] || get_part(part)
-        keys.each{|key| value = value[camelize key]}
+        keys.each{|key| value = value[camelize key]} if value
```

Comment:
```rb
keys.each do |key|
  value = value[camelize key] if value
end
```

Reply:
Well, they're slightly different, right? I think it depends on how we want the library to behave when there are nested keys that may or may not be there.